Tests/MQTTnet.Core.Tests/AsyncQueue_Tests.cs

```diff
@@ -120,22 +120,5 @@ public void Dequeue_Sync()
             Assert.AreEqual("2", queue.TryDequeue().Item);
             Assert.AreEqual("3", queue.TryDequeue().Item);
         }
-
-        [TestMethod]
-        public void Clear()
```

Comment:
Why is this test deleted? In my opinion this should still work.

Reply:
Because Clear() was deleted.